back-end/models/Table.js

```diff
@@ -4,46 +4,87 @@ const Schema = mongoose.Schema;
 
 // Create Schema
 const TableSchema = new Schema({
-  location: {
+
+  ID:{
+    type: Number,
+    required:true
+
+  },
+  locationName: {
     type: PointSchema,
     required: true
   },
-  name: {
+  streetAddress: {
     type: String,
-    required: true
+    required: false
   },
-  description: {
+  city: {
     type: String,
-    required: true
+    required: false
   },
-  status: {
+
+  state: {
     type: String,
-    enum: ['submitted', 'approved', 'deleted', 'reported'],
-    default: 'submitted'
+    required: false
   },
-  tableType: {
+  zipcode: {
     type: String,
-    enum: ['fixed-height', 'adjustable', 'portable'],
-    required: true
+    required: false,
+    max:5
+  },
+
+  locationWithinBuilding: {
+    type: String,
+    required: false
   },
   restroomType: {
     type: String,
     enum: ['men', 'women', 'family', 'other'],
+    required: false
+  },
+  coordinateLocation: {
+    type: Array,
```

Comment:
I believe we want this to be type Point: https://www.mongodb.com/docs/manual/reference/geojson/